openrtb_ext/imp.go

```diff
@@ -84,3 +86,12 @@ type ExtStoredBidResponse struct {
 type Options struct {
 	EchoVideoAttrs bool `json:"echovideoattrs"`
 }
+
+// GetImpIDs returns slice of all impression Ids from impList
+func GetImpIDs(impList []openrtb2.Imp) []string {
+	impIDs := []string{}
```

Comment:
Nitpick: Please rename `impList` to `imps`.

Reply:
Thanks @SyntaxNode for this.
Done.